### PR TITLE
Add basic validation

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: src/views/Home/utils.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        /*
+        parentName: isDefined(pointer.parentName)
+            ? (feature.properties[pointer.parentName]) as string | undefined
+            : undefined,
+        */
+    }));
+}
+
+function validateSettings(settings: Settings, root = false): Error[] {
+    function getErrors(
```
Comment (AdityaKhatri) [inline src/views/Home/utils.ts:49]: why is this inside?

File: src/views/Home/utils.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            : undefined,
+        /*
+        parentName: isDefined(pointer.parentName)
+            ? (feature.properties[pointer.parentName]) as string | undefined
+            : undefined,
+        */
+    }));
+}
+
+function validateSettings(settings: Settings, root = false): Error[] {
```
Comment (AdityaKhatri) [inline src/views/Home/utils.ts:48]: What is root here?
Please leave a note for future reference as well

File: src/views/Home/utils.ts (hunk 1 above)
Comment (tnagorra) [reply src/views/Home/utils.ts:49]: Because this is not used anywhere else. Private function!

Comment (AdityaKhatri) [reply src/views/Home/utils.ts:49]: humm, not a big fan of private functions.
We should share.

**Pull request merged**